Makefile

```diff
@@ -1,6 +1,7 @@
 -include awx/ui_next/Makefile
 
-PYTHON := $(notdir $(shell for i in python3.9 python3; do command -v $$i; done|sed 1q))
+# use alias or update-alternatives to force specific version on your sytem or shell
+PYTHON := python3
```

Comment:
This will take whatever `python3` bin or shim or link is sitting around. If you want to upgrade to 3.11, then instead change Line 3: s/python3.9/python3.11/g